client/blocks/login/index.jsx

```diff
@@ -46,6 +47,12 @@ class Login extends Component {
 		}
 	};
 
+	componentWillReceiveProps = ( nextProps ) => {
+		if ( this.props.twoFactorAuthType !== nextProps.twoFactorAuthType ) {
```

Comment:
Why don't we use global notices in case of errors? It was discussed several times in the past and there was decision made to use global notices to avoid scrolling of the page to show messages.